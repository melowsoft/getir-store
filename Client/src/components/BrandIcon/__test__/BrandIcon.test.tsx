import React from "react"
import  ReactDOM  from "react-dom"
import { BrandIcon } from ".."
import { cleanup } from '@testing-library/react';
import renderer from "react-test-renderer"



afterEach(cleanup)

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<BrandIcon ></BrandIcon>, div);
  ReactDOM.unmountComponentAtNode(div)
})


it("matches snapshot", () => {
  const tree = renderer.create(<BrandIcon></BrandIcon>).toJSON()
  expect(tree).toMatchSnapshot()
})

