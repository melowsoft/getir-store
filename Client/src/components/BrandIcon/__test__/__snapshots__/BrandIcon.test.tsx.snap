// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="sc-bdnxRM gcpWuX"
  data-testid="brand-logo"
>
  <img
    alt="brand-logo"
    className="sc-gtsrHT ldihJO"
    src="Logo.svg"
  />
</div>
`;
